src/index.html

```diff
@@ -1,12 +1,430 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a class="logo" href="#Home">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link nav__link--active is-active" href="#Apple">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Samsung">Samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Smartphones">Smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              data-qa="nav-hover"
+              href="#Laptops&Computers"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#Video">Video</a>
+          </li>
+        </ul>
+      </nav>
+
+    </header>
+    <main class="main-content page__main">
+      <div class="card" data-qa="card">
+        <img
+          class="card__image"
+          src="./images/imac.jpeg"
+          alt="apple computer"
+        >
+
```

Comment:
I think you should add one more container like this for all the cards
```
<main. class="main-content page__main">
   <div class="cards"> (add this container as a parent to all 8 cards)
      <div class="card" data-qa="card">
```

and consider this checklist point , 
```
there should always be 4 cards in a row (not 2, 3 or 5)
use :nth-child(4n) to select every 4th element
use :nth-last-child(-n + 4) to select 4 last elements
```

Reply:
> I think you should add one more container like this for all the cards

Done. It doesn't change the test results though. What purpose does it serve?

> there should always be 4 cards in a row (not 2, 3 or 5)
> use :nth-child(4n) to select every 4th element
> use :nth-last-child(-n + 4) to select 4 last elements

Not sure what I'm trying to achive here? Constant number of .card elements in a row is already defined by container width. Also I used :nth-child(4n) selector.